app/views/account/contacts/index.html.slim

```diff
@@ -0,0 +1,41 @@
+.container
+  .row
+    .col-sm
+      h1#contact_header Contacts from users
+      = paginate @contacts
+      = search_form_for @q, url: account_contacts_path do |f|
+        = f.label :name_cont
+        = f.search_field :name_cont
+        = f.label :email_cont
+        = f.text_field :email_cont
+        = f.label :phone_cont
+        = f.text_field :phone_cont
+        = f.submit "Search", class: 'btn btn-primary'
+      .error_message
```

Comment:
usually form errors displayed before inputs (at form top)

Reply:
Yes, but this div for other purpose. Table below has links with remote:true helper and ajax errors raises here.